Classes/NKJPagerViewController.m

```diff
@@ -338,6 +343,11 @@ - (void)scrollViewDidScroll:(UIScrollView *)scrollView
     }
 }
 
+- (void)scrollViewDidEndDecelerating:(UIScrollView *)scrollView
```

Comment:
I think that there is a case of `scrollViewDidEndScrollingAnimation` when scrollview's animation finish!